src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1025,11 +1020,6 @@ private static BoundExpression MakeLiteral(SyntaxNode syntax, ConstantValue cons
                     {
                         var value = tempStores[correspondingStore].Right;
 
-                        // When we created the temp, we dropped the argument RefKind
-                        // since the local contained its own RefKind. Since we're removing
-                        // the temp, the argument RefKind needs to be restored.
-                        refKinds[a] = ((BoundLocal)argument).LocalSymbol.RefKind;
```

Comment:
The old comment is wrong. We do not drop ref kinds when creating temps (maybe we did long time ago...).
 Also refKind of the temp loses `out` vs. `ref` distinction possibly resulting in incorrect ref kind set here.